front/src/components/business/Filter/Filter.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { FilterItem, FilterItemProps } from './FilterItem';
+import { Typography } from '@/components/common/Typography';
+
+export type FilterProps = {
+  title: string;
+  filters: FilterItemProps[];
+};
+
+export const Filter = ({ title, filters }: FilterProps) => {
+  return (
+    <div>
+      <Typography color="black" size="text-xs">
+        {title}
+      </Typography>
+
+      <div className="mt-2.5 flex flex-wrap gap-2">
+        {filters.map((item, index) => (
+          <FilterItem key={index} title={item.title} />
```

Comment:
ici le mieux c'est de faire : 

```suggestion
          <FilterItem key={index} {...item} />
```

en fait ça permet de passer tous les params au fils qui attends un object de type FilterItemProps